src/components/Main/Main.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useContext } from 'react';
+import { TodoList } from '../TodoList/TodoList';
+import { DispatchContext, TodosContext } from '../../Store';
+import { Todo } from '../../Types/Todo';
+
+export const Main: React.FC = () => {
+  const {
+    todos,
+    todosCompleted,
+    selectedActive,
+    selectedCompleted,
+  } = useContext(TodosContext);
+  const dispatch = useContext(DispatchContext);
+
+  const handleChange = () => {
+    dispatch({ type: 'todosCompleted' });
+
+    if (!todosCompleted) {
+      dispatch({ type: 'changeTodosCompletedTrue' });
+    }
+
+    if (todosCompleted) {
+      dispatch({ type: 'changeTodosCompletedFalse' });
+    }
+  };
+
+  const filterTodos = (items: Todo[]) => {
+    if (selectedActive) {
+      const activeTodos = items.filter(todo => todo.completed === false);
+
+      return activeTodos;
+    }
+
+    if (selectedCompleted) {
+      const activeTodos = items.filter(todo => todo.completed === true);
```

Comment:
You have a typo in the variable name. It should be 'completedTodos' instead of 'activeTodos'. Please fix this to maintain the correct functionality